internal/fourslash/test_parser.go

```diff
@@ -75,8 +105,12 @@ func ParseTestData(t *testing.T, contents string, fileName string) TestData {
 		markers = append(markers, file.markers...)
 		ranges = append(ranges, file.ranges...)
 		for _, marker := range file.markers {
-			if _, ok := markerPositions[marker.Name]; ok {
-				t.Fatalf("Duplicate marker name: %s", marker.Name)
+			if marker.Name == "" && marker.Data != nil {
```

Comment:
I fixed this in https://github.com/microsoft/typescript-go/pull/1357/ by having a marker name be `*string` instead of `string`, since in Strada, so probably no need for this change now except for the improved error message.